perma_web/replay/views.py

```diff
@@ -27,7 +27,7 @@ def iframe(request):
         context['target_url'] = link.screenshot_capture.url if screenshot else link.submitted_url
         context['embed_style'] = 'replayonly' if replay_only else 'default'
         context['web_worker'] = web_worker
-        context['sandbox'] = link.primary_capture.use_sandbox()
+        context['sandbox'] = link.primary_capture.use_sandbox() and not ondemand
```

Comment:
What do you think of maybe adding a comment here so we remember more easily why we added this in the future?

Reply:
Yes, I should, will do

Reply:
Actually I am rethinking the location this logic and comment..... probably I should alter https://github.com/harvard-lil/perma/blob/e761f4ab597c293f0fd6612e015428100899f28b/perma_web/perma/models.py#L1928 to 

` return not self.mime_type().startswith("application/pdf") and not self.show_interstitial()`